src/VisualStudio/Core/Def/Implementation/Preview/FileChange.cs

```diff
@@ -79,13 +80,18 @@ private ChangeList ComputeChildren(TextDocument left, TextDocument right, Cancel
 
             diffService = diffService ?? diffSelector.DefaultTextDifferencingService;
 
-            var diff = ComputeDiffSpans(oldText, newText, diffService, cancellationToken);
-            if (!diff.Differences.Any())
+            var diff = ComputeDiffSpans(diffService, left, right, cancellationToken);
+            if (diff.Differences.Count == 0)
             {
-                // There are no changes. Just show the whole document.
-                return GetEntireDocumentAsSpanChange(left);
+                // There are no changes.
+                return ChangeList.Empty;
```

Comment:
see bug TFS - 1037274

ya, I am not sure why or who decided to show whole document when there is no change. now, when there is no change, it shows no change.